Wallet testing in Ci and calibnet ci checks to a script (#2376)
ChainSafe/forest: documentation/src/cli.md

```

When the Forest daemon is started, an admin token will be displayed (alternatively, use `--save-token <token>` flag to save it on disk).

> ## Sending filecoin tokens (money) from your wallet.

Reviewer: I wouldn't associate tokens with money.
```suggestion
## Sending Filecoin tokens from your wallet
```

Reviewer 2: I think so too.

> For sending filecoin token, forest daemon must be running. You can do so by running:

Reviewer: ```suggestion
For sending Filecoin tokens, the Forest daemon must be running. You can do so by running:
```

> `forest --chain calibnet --target-peer-count 50 --encrypt-keystore false`

Reviewer: Do we need this peer count and disable the keystone? I mean, it's OK for the CI and testing, but in general, we should recommend and default to have it encrypted.

> Next, to send filecoin token (money) to a wallet address:

Reviewer: ```suggestion
Next, send Filecoin tokens to a wallet address:
```

`forest-cli --token <admin_token> send <wallet-address> <amount in attoFIL>`

> where 1 attoFIL = 10e−18 FIL.

Reviewer: It's not a pretty representation and I believe it's not accurate - 10e-18 is 10 attoFIL, no? Let's utilize some markdown capabilities:
```suggestion
where 1 attoFIL = $10^{−18}$ FIL.
```

## Wallet

> Filecoin wallets are stored under forest data directory (e.g., ~/.local/share/forest in case of linux) in a `keystore` file.

Reviewer: ```suggestion
Filecoin wallets are stored under the Forest data directory (e.g., `~/.local/share/forest` in the case of Linux) in a `keystore` file.
```

> All wallet commands require write permissions and admin token (`--token`) to interact with the keystore.
> The admin token can be retrieved from forest startup logs.

Reviewer: ```suggestion
All wallet commands require write permissions and an admin token (`--token`) to interact with the keystore.
The admin token can be retrieved from forest startup logs or by running it with `--save-token <PATH>`.
```

to get formatted keys for importing. Usage:
`forest-cli wallet import <hex encoded json key>`

> <<<<<<< HEAD Export Export a key by address. Use a wallet address to export a

Reviewer: fiasco
